dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -12,90 +12,18 @@
   %i.fa.fa-chevron-left
   %span= t('crud.back')
 
-%h2= @page_title
+%h1.text-black= @page_title
```

Comment:
Should this title be moved inside the React component?

Reply:
I'm kind of torn on this as well... I think eventually we'll want a full page container react component that holds all the react component sections. we'd really be able to slim down edit.html.haml and edit.js to just mount the new page component. I picture the h1 and back button being rendered there. this was just to get it a little closer to the full page design, but yeah, it's half baked at the moment

Reply:
Sounds good! Yeah either way on this is totally good